test/components/Container.spec.js

```diff
@@ -16,7 +16,7 @@ import ComponentStore from 'src/helpers/componentStore';
 import Constants from 'src/constants';
 import sinon from 'sinon';
 import { Map as immutableMap } from 'immutable';
-
+import * as ExecuteEvents from '../../src/helpers/ExecuteEvents';
```

Comment:
can you import as `import { executeEventsFromCurrentRecord } from '../helpers/ExecuteEvents';? Just as in 
**src/components/Container.jsx**